release/ray_release/reporter/db.py

```diff
@@ -45,6 +43,20 @@ def report_result(self, test: Test, result: Result):
                 Record={"Data": json.dumps(result_json)},
             )
         except Exception:
-            logger.exception("Failed to persist result to the databricks delta lake")
+            try:
+                # This may happen if metrics are too big.
+                # TODO persist big metrics in an alternative fashion
+                logger.warning(
+                    "Couldn't persist with prometheus_metrics, trying without them"
+                )
+                result_json.pop("prometheus_metrics", None)
```

Comment:
Should we set this to an empty dict for schema compatibility? Or is this auto filled by databricks?

```suggestion
                result_json["prometheus_metrics"] = {}
```

Reply:
it will be auto set to NULL